guillotina_amqp/amqp.py

```diff
@@ -68,7 +74,13 @@
     if name in connections:
         connection = connections[name]
         return connection["channel"], connection["transport"], connection["protocol"]
-    channel, transport, protocol = await connect()
+    try:
+        channel, transport, protocol = await connect()
```

Comment:
`connect()` is only called at start so if this error happens, there will not be anything in the connections cache to remove actually right?

Maybe we also just need `@backoff` here and attempt to retry on this exception?

This is my bad. I thought this error was manifesting somewhere else.

Reply:
:+1: